.github/workflows/push-job-sched-jar.yml

```diff
@@ -0,0 +1,47 @@
+name: Upload Job-Scheduler Jar to Maven
+
+on:
+  push:
+    tags: 
+      - v*
+jobs:
+  upload-job-scheduler-jar:
+    runs-on: [ubuntu-16.04]
+    strategy:
+      matrix:
+        java: [12]
+    name: Upload Job-Scheduler Jar to Maven
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v1
+        
+      - name: Configure AWS CLI
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
```

Comment:
Suggest removing the matrix setup since we don't actually use it

Reply:
Are you sure we don't need to setup a specific version of java?

Reply:
removed java matrix as per our call.